packages/ocamlapi_ppx/ocamlapi_ppx.0.0.1/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+maintainer: "stephanos.tsoucas@gmail.com"
+authors: ["Stephanos Tsoucas"]
+license: "Apache-2.0"
+homepage: "https://github.com/nosman/Ocamlapi"
+synopsis: "Path-based HTTP request routing for Ocaml"
+bug-reports: "https://github.com/nosman/Ocamlapi/issues"
+dev-repo: "git+https://github.com/nosman/Ocamlapi.git"
+build: [
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+ 
+depends: [
+  "ocaml" {>= "4.04"}
+  "dune"
+  "ocamlapi"
```

Comment:
here you probably want specifically `ocamlapi.0.0.1` and not any versions. (same with the other packages)